cohort-engine/src/main/java/com/ibm/cohort/engine/LoggingEnum.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ *  * (C) Copyright IBM Corp. 2021
+ *  *
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.ibm.cohort.engine;
+
+public enum LoggingEnum {
+	COVERAGE("coverage"),
+	TRACE("trace"),
+	NA(null);
+
+	private String debugLevel;
+
+	LoggingEnum(String debugLevel){
+		this.debugLevel = debugLevel;
+	}
+
+	public static LoggingEnum getLoggingFromString(String value){
+
+		try{
+			return LoggingEnum.valueOf(value);
+		}
+		catch(Exception e) {
+			return LoggingEnum.NA;
+		}
+	}
```

Comment:
Pedantic: I don't see this method being used anywhere.  I'm OK with keeping it around if we think people will want to do a "parse value with default" operation in the future.